<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonalTask</class>
 <widget class="QWidget" name="PersonalTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 lightgreen, stop:1 darkgreen);
}
</string>
  </property>
  <widget class="QLabel" name="Workdate">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Due Date:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="AddPriorityPersonal">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>High </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Medium</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Low</string>
    </property>
   </item>
  </widget>
  <widget class="QDateTimeEdit" name="AddDatePersonal">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background: white; /* Set the background color */
    color: black;      /* Set the text color */
    border: 2px solid #56ab2f; /* Add a green border */
    border-radius: 5px; /* Optional: Rounded corners */
    padding: 5px; /* Padding for better spacing */
}
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddPersonalButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Personal Task</string>
   </property>
  </widget>
  <widget class="QLabel" name="PersonalPrority">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Priority:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddTitlePersonal">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background: white; /* Set the background color */
    color: black;      /* Set the text color */
    border: 2px solid #56ab2f; /* Add a green border */
    border-radius: 5px; /* Optional: Rounded corners */
    padding: 5px; /* Padding for better spacing */
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="PersonalTasklabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:700; font-style:italic;&quot;&gt;Personal Task&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BacktoTaskfromPersonal">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Back to Task Manager</string>
   </property>
  </widget>
  <widget class="QLabel" name="PersonalTaskTitle">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Task Title:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
